tests/deploy_test.go

```diff
@@ -13,6 +13,42 @@ import (
 
 var emptyQueryString = ""
 
+const secret = "tDsdf7sFT45gs8D3gDGhg54"
```

Comment:
If this is meant to come from outside the scope of the Go code, can you perhaps read it in via environmental variable?